.eslintrc.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
 		'@wordpress/valid-sprintf': 'warn',
 		'jsdoc/check-tag-names': [
 			'error',
-			{ definedTags: [ 'jest-environment', 'hook' ] },
+			{ definedTags: [ 'jest-environment', 'filter', 'slotFill' ] },
```

Comment:
Nice, I like the increased specificity here.  I don't think we have any in WCA yet, but would it be worth it to add `action` here as well since these could be added at some point?